fixtures/setup.json

```diff
@@ -1,23 +1,35 @@
 {
   "seed": 4,
   "keysToGen": 5,
+  "importKeys": [{"privateKey":"/8UoXvzGIU2H/TgTAmswx96m2dVCXkRqnIIlDhJrGCo=","sigType":2}],
```

Comment:
Move this line after preAllocGenKeys since the doc elsewhere says these keys come after the gen keys. Or change so that these keys are ordered first.

Reply:
I've renamed the other field to `preallocatedFunds` to clarify it doesn't actually generate the keys and that it applies to both generated and imported keys. Given that, I think ordering makes the most sense the way it is.